source/linear_solver_parameters.cc

```diff
@@ -0,0 +1,538 @@
+/*
+ * linear_solver_parameters.cc
+ *
+ *  Created on: May 5, 2021
+ *      Author: sg
+ */
+
+#include <rotatingMHD/linear_solver_parameters.h>
+
+#include <deal.II/base/conditional_ostream.h>
+
+namespace RMHD
+{
+
+/*!
+ * @brief Namespace containing all the structs and enum classes related
+ * to the run time parameters.
+ */
```

Comment:
Documentation of namespaces, classes and methods should all be in the header files

Reply:
Fixed in 58843d9.